### DISCUSSION TITLE
Where is the user data for Navidrome stored?

### DISCUSSION THREAD

**Discussion by Rycia** [navidrome/navidrome#2314] [Q&A]
I'd like to move Navidrome to a different PC and set up backups, but I can't find where the user data is stored. Where is user data stored? I've moved Navidrome PC to PC before but in that process I've lost my data because it doesn't seem stored within Navidrome's data itself and I can't find it anywhere in %appdata% or program files. Running off a Windows 10 machine.

**Comment by Rycia** [comment]
Bump. Still wanting an answer.

**Comment by deluan** [comment]
You just have to move the `navidrome.db*` files found in the `DataFolder`.  Can you share your configuration so I can be more assertive?
